docs/clustering.md

```diff
@@ -335,6 +335,9 @@ It's important to note that not all instances are required to be provisioned wit
 Project updates behave differently than they did before. Previously they were ordinary jobs that ran on a single instance. It's now important that
 they run successfully on any instance that could potentially run a job. Project's will now sync themselves to the correct version on the instance immediately
 prior to running the job.
+When the sync happens, it is recorded in the database as a project update with a `launch_type` of "sync"
+and a `job_type` of "run". Project syncs will not change the status or version of the project,
+but will, instead, update the source tree only on the instance where they run.
```

Comment:
* Unless it is the first project update ran. Said differently, unless there are no other previous project updates, of any launch_type or job_type, for the project.

Reply:
valid to include, I added a blurb about it.